LAB 3.ipynb

```diff
@@ -0,0 +1,481 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$\\int_{\\Delta x_0}^{\\infty} \\frac {1}{\\sqrt {2\\pi \\sigma ^{2}}}e^{-{\\frac {(x-\\mu )^{2}}{2\\sigma ^{2}}}} = \\frac{1-\\mathrm{erf} \\biggl(\\frac{(\\Delta x_0 - \\mu)}{\\sqrt{2}\\; \\sigma}\\biggr)}{2} = \\frac{1-\\mathrm{erf} \\biggl(\\frac{(\\chi - \\mu)}{\\sqrt{2}\\; \\sigma}\\biggr)}{2}$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$\\mathrm{erf} \\biggl(\\frac{(\\Delta x_0 - \\mu)}{\\sqrt{2}\\; \\sigma}\\biggr) = \\mathrm{erf} \\biggl(\\frac{(\\chi - \\mu)}{\\sqrt{2}\\; \\sigma}\\biggr)$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$\\Delta x_0 = \\chi \\sigma$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Setting $\\chi$ equal to 5, $\\mu$ equal to 0, and $\\sigma$ equal to 1 we can find what our $\\Delta x_0$ is when $\\Delta y =0$."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$\\boxed{ \\Delta x_0 = 5 \\text{ arcseconds}}$$\n",
+    "\n",
+    "$$\\boxed{ \\sqrt{\\Delta x^2 + \\Delta y^2} \\geq 5 \\text{ arcseconds}}$$\n",
+    "\n",
+    "$$\\text{This is where we would say we detected an asteroid}$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Problem 3:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Question in Words: \n",
+    "We expect the moon to effectively shadow the Earth from cosmic $\\gamma$ rays. In a moon sized patch of the sky we expect to see 1 $\\gamma$ ray a minute and this effectively gives us a Poisson background pdf. Assuming a Poisson signal-free background pdf with $\\mu$ equal to 1 cosmic ray per minute, what is the significance of detecting 6,800 counts of $\\gamma$ rays after 7,200 minutes of observing the moon?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In math: This is a number of trials problem. So what we need to do is find our background pdf for a single minute and then multiply that by the 7,200 minutes of observing the moon to get the proper background pdf. Then to get the probability that we observe 6,800 or less counts of $\\gamma$ rays we would integrate our proper background pdf from -$\\infty$ to 6,800. Then we find what value of sigma that occurs for a standard normal distribution."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$\\mathrm{pdf}_{1minute}(k) = \\frac{e^{-\\mu}\\mu^k}{k!} = \\frac{1}{ek!}$$\n",
+    "\n",
+    "$$\\mathrm{Proper\\ pdf}(k)= \\frac{7200}{ek!} $$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$P(k \\leq 6800) = 7200 \\cdot \\int_{-\\infty}^{6800} \\mathrm{pdf}_{1minute}(k) dk =7200 \\cdot \\int_{-\\infty}^{6800} \\frac{e^{-\\mu}\\mu^k}{k!} dk = \\int_{-\\infty}^{6800} \\frac{7200}{ek!}dk$$"
```

Comment:
your integral here should be from -infinity to X. It doesn't make sense that you would integrate the upper end of the gaussian distribution in this case

Reply:
This makes sense because that integral is the survival function. I am using it to find the chi value from the Gaussian. I could go from - infinity to -chi but that would just be the cdf function and equivalent.